lib/models/lookup.js

```diff
@@ -58,7 +57,7 @@ function LookupModelFactory (
             },
             {
                 keys: { ipAddress: 1},
-                options: { unique: true }
+                options: { unique: false }
```

Comment:
@cgx027 : I checked @sunnyqianzhang 's system, and found multiple lookup entries don't have ipAddress (ipAddress is not a required item for a lookup entry). So this is the reason why the ipAddress index need not be unique.